recipes/ptxcompiler/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "ptxcompiler" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/rapidsai/ptxcompiler/archive/refs/tags/{{ version }}.tar.gz
+  sha256: c7021a0946478404c4055a612fd725af95d53ec23eb3dd8d8bc0b5ac55eb44d0
+  fn: ptxcompiler.tar.gz
+
+build:
+  number: 0
+  skip: true  # [not linux]
+  skip: true  # [cuda_compiler_version == "None"]
+  skip: true  # [cuda_compiler_version == "10.2"]
+  skip: true  # [cuda_compiler_version == "11.0"]
+  script_env: 
+    - CUDA_HOME  # [ppc64le or aarch64]
```

Comment:
```suggestion
```

I'm really confused why `CUDA_HOME` is being added at this stage. As far as conda-forge is concerned, there are no cuda builds with `ppc64le` or `aarch64` therefore, they would all be skipped.